client/app/scripts/superdesk/filters.js

```diff
@@ -116,5 +118,46 @@ define([
 
                 return merged.join(', ');
             };
-        });
+        })
+        .filter('formatDatelinesDate', function() {
+            return function(located, date_to_format) {
+                var momentizedTimestamp = angular.isDefined(date_to_format) ? moment.utc(date_to_format) : moment.utc();
+
+                if (angular.isDefined(located) && located.tz !== 'UTC') {
+                    momentizedTimestamp = momentizedTimestamp.tz(located.tz);
+                }
+
+                if ((momentizedTimestamp.month() + 1) >= 3 && (momentizedTimestamp.month() + 1) <= 7) {
+                    return momentizedTimestamp.format('MMMM DD');
```

Comment:
This will not format the month September to "Sept". Maybe worth checking with POs?